components/TopBar/TopBar.tsx

```diff
@@ -1,11 +1,39 @@
-import { ThemeToggler } from "../ThemeToggler/themeToggler"
+import { useRouter } from "next/router";
+import { IconContext } from "react-icons";
+import { FaGithub, FaSlackHash } from "react-icons/fa";
+import { ThemeToggler } from "../ThemeToggler/themeToggler";
 
-export const TopBar = () => {
-    return (
-        <div className="flex justify-end fixed w-[100%] md:sticky">
-            <span className="mx-10 py-5">
-                <ThemeToggler/>
-            </span>
-        </div>
-    )
-}
\ No newline at end of file
+export const TopBar = ({ header }: { header?: string }) => {
+  const router = useRouter();
+  const category = router.asPath.replace("/", "");
+  return (
+    <div className="flex items-center justify-between mb-4">
+      <div className="flex items-center text-3xl my-4 dark:text-gray-300">
+        <FaSlackHash className="mt-1 mr-2 text-gray-600 dark:text-gray-300" />
+        <span className="text-gray-900 dark:text-gray-100 uppercase font-bold">
+          {header ?? category}
+        </span>
+      </div>
+      <div className="items-center flex justify-center gap-2">
```

Comment:
reduce this gap a bit, 
![image](https://user-images.githubusercontent.com/78981177/213933946-40c3e4ad-e8db-4bb7-b5f1-5f4a79bb42f4.png)